pandas/tests/tseries/offsets/test_offsets.py

```diff
@@ -769,6 +775,14 @@ def test_constructor_errors(self):
             BusinessHour(start='AAA')
         with pytest.raises(ValueError):
             BusinessHour(start='14:00:05')
+        with pytest.raises(ValueError):
+            BusinessHour(start=[])
+        with pytest.raises(ValueError):
+            BusinessHour(start=['09:00', '11:00'])
+        with pytest.raises(ValueError):
+            BusinessHour(start=['09:00', '11:00'], end=['10:00'])
+        with pytest.raises(ValueError):
+            BusinessHour(start=['09:00', '11:00'], end=['12:00', '20:00'])
```

Comment:
Can you update the `pytest.raises` calls (for this entire test function) to include a check for the error message?  Just pass in `match` parameter that indicates the expected message.

Reply:
Updated